### ISSUE TITLE
Handle -inf for Torchscript of FX graphs

### ISSUE THREAD

**Issue by anijain2305** [facebookresearch/torchdynamo#124]
Repro

~~~
import torch
import torch.fx

x = torch.randn(4, 5)
mask = torch.randn(4, 5) > 0.5

def f(x, mask):
    # return x.masked_fill_(mask, 1.0) # PASSES
    return x.masked_fill_(mask, float("-inf"))

print(f(x, mask))

# Only fails when symbolic_trace
fx_mod = torch.fx.symbolic_trace(f)
scripted_f = torch.jit.script(fx_mod)
print(scripted_f(x, mask))
~~~

@eellison
